templates/page/campus/home.html.twig

```diff
@@ -73,24 +73,16 @@
                 <div class="campus__text campus__text--center">
                     <h2>Passer à l'action</h2>
                     <p>
-                        Inscrivez-vous vite pour participer à notre première formation
-                        collective qui a pour thème "Agir près de chez moi".
+                        Nous proposons enfin des formations collectives en ligne sur plusieurs semaines, communément appelées MOOC ("Massive Open Online Courses").
```

Comment:
Please split it in several lines to increase readability.
A line _should not_ exceed 80 characters.

Like @thomasbisignani  once said:
> scrolling horizontally is something i wouldn't wish on my worst enemy 👌

Reply:
ok : )